app/sitemap.py

```diff
@@ -15,142 +15,74 @@
 
 """Exports the URLs of all person entries to a sitemap.xml file."""
 
-__author__ = 'jocatalano@google.com (Joe Catalano) and many other Googlers'
-
 import logging
+import requests
+import requests_toolbelt.adapters.appengine
+import time
+import urllib
 
 from datetime import datetime, timedelta
-from google.appengine.api import urlfetch
-from model import *
-from time import *
-from utils import *
-
-def _compute_max_shard_index(now, sitemap_epoch, shard_size_seconds):
-    delta = now - sitemap_epoch
-    delta_seconds = delta.days * 24 * 60 * 60 + delta.seconds
-    return delta_seconds / shard_size_seconds
-
-def _get_static_sitemap_info(repo):
-    infos = StaticSiteMapInfo.all().fetch(2)
-    if len(infos) > 1:
-        logging.error("There should be at most 1 StaticSiteMapInfo record!")
-        return None
-    elif len(infos) == 1:
-        return infos[0]
-    else:
-        # Set the sitemap generation time according to the time of the first
-        # record with a timestamp.    This will make the other stuff work
-        # correctly in case there is no static sitemap.
-        query = Person.all_in_repo(repo)
-        query = query.filter('last_modified != ', None)
-        first_updated_person = query.order('last_modified').get()
-        if not first_updated_person:
-            # No records; set the time to now.
-            time = get_utcnow()
-        else:
-            # Set the time to just before the first person was entered.
-            time = first_updated_person.last_modified - timedelta(seconds=1)
-        info = StaticSiteMapInfo(static_sitemaps_generation_time=time)
-        db.put(info)
-        return info
+from google.appengine.api import taskqueue
+
+import const
+from model import Repo
+from utils import BaseHandler
+
+
+# Use the App Engine Requests adapter. This makes sure that Requests uses
+# URLFetch.
+# TODO(nworden): see if we should condition this on the runtime (Python 2 vs. 3)
```

Comment:
Did you mean to leave this TODO in here?

Reply:
Yeah.